src/biokbase/narrative/app_util.py

```diff
@@ -1,6 +1,8 @@
+"""App-related utility functions."""
+
 import json
 import re
-from typing import Any, Optional
+from typing import Any
```

Comment:
The changes in this file comprise:
- adding type info
- whitespace/wrapping changes
- converting `"string {} ".format(whatever)` to `f"string {whatever}"`
- a couple of list comprehensions in place of iteration